packages/teams-js/src/internal/utils.ts

```diff
@@ -58,6 +62,10 @@ export function validateOrigin(messageOrigin: URL): boolean {
     }
   }
 
+  validateOriginLogger(
+    'Origin %s is invalid because it is not an origin approved by this library or the call to app.initialize',
```

Comment:
>an origin approved by this library [](http://example.com/codeflow?start=44&length=34)

Not sure if this is the place to do it, but could we/do we dump the valid origins to the log stream at some point to aid in debugging?

Reply:
No problem. Also added the list of origins that were passed to app.initialize